front/lib/http_utils.ts

```diff
@@ -49,3 +49,13 @@ export function parse_payload<SchemaType>(
     new RequestParseError(validate.errors?.map((e) => e.message).join(", "))
   );
 }
+
+export function parseServerSentEventChunk(chunk: Uint8Array): any | null {
```

Comment:
Should we use `see` here instead? I don't believe you have guarantee that the chunk respects the event boundary.